### DISCUSSION TITLE
Incorrect derivation path for first Komodo address (KMD)

### DISCUSSION THREAD

**Discussion by loupiote** [LedgerHQ/ledger-live-desktop#4006] [Coin/Token request]
#### Ledger Live Version and Operating System

LL 2.23 win10

#### Expected behavior

Create a Komodo address.

The first Komodo address should have the derivation path m/44'/141'/0'/0/0

#### Actual behavior

The first Komodo address uses the derivation path m/44'/141'/0'/0/1 

#### Steps to reproduce the behavior

Ledger Live > Add Account > Komodo (KMD)

Then look at the derivation path in the account details > advanced.

**Comment by cthepot** [comment]
Please try the latest version. If it does not solve your situation please contact support on support.ledger.com.
